src/all/baobua/src/eu/kanade/tachiyomi/extension/all/baobua/BaoBua.kt

```diff
@@ -0,0 +1,111 @@
+package eu.kanade.tachiyomi.extension.all.baobua
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.util.asJsoup
+import keiyoushi.utils.tryParse
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+class BaoBua() : SimpleParsedHttpSource() {
+
+    override val baseUrl = "https://www.baobua.net"
+    override val lang = "all"
+    override val name = "BaoBua"
+    override val supportsLatest = true
```

Comment:
It is assumed sites have a popular listing always, in this case it appears to only have a latest listing. By convention the latest listing has been put in the popular tab if it is the only one available.
```suggestion
    override val supportsLatest = false
```